src/main/java/com/alevel/backend/domain/Alchol.java

```diff
@@ -0,0 +1,66 @@
+package com.alevel.backend.domain;
+
+import lombok.*;
+
+import javax.persistence.*;
+
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Entity
+public class Alchol{
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private int id;
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

javax.persistence.Id annotation should not be attached to 32-bit data. Using 32-bit data types is risky because loading from the database will cause an exception when the table passes 2^32 entries. A mix of 32-bit and 64-bit values can lead to integer wrap-arounds. Consider monitoring the growth of the relevant tables to estimate when integer overflow would happen. Consider changing the field annotated with @Id from integer to long. Also consider adding an alarm that notifies you when the number of rows increases by an order of magnitude.